src/main/java/com/philderbeast/prisonpicks/Pick.java

```diff
@@ -23,12 +24,14 @@
     static final String FORTUNE = "fortune";
     static final String UNBREAKING = "unbreaking";
     static final String SILK_TOUCH = "silktouch";
+    
+    int rng = (int) (Math.random() * (4 - 1)) + 1;
```

Comment:
why is this set once for the class?????????

Reply:
for the chance to display a different break message

Reply:
this value is static though, it should be in the method, otherwise its set when its first loaded and never again